index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/70b9059057.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="tool-bar">
+      <h2 class="logo">John Doe</h2>
+      <i class="fa-solid fa-bars fa-2x"></i>
+    </header>
+    <div class="headline">
```

Comment:
Please change this `div` tag to a `section` tag because It represents a generic standalone section of the document, [check this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info